CHANGELOG.md

```diff
@@ -86,13 +167,9 @@
 
 **Fixed bugs:**
 
-- chef install doesn't seem to use the Default SSL Policy [\#488](https://github.com/chef/chef-dk/issues/488)
```

Comment:
Or maybe remove `Upstream Bug` from the list of ignored flags - https://github.com/chef/chef-dk/issues/308